### DISCUSSION TITLE
How to add a CMS like Strapi

### DISCUSSION THREAD

**Discussion by 28development** [micheleriva/krabs#19] [General]
Hi, not an "issue", but I was wondering how one should add a cms like strapi to an next app using krabs, since the app already contains a backend with express.
Or does it not make sense to use this and headless cms like strapi?

**Comment by 28development** [comment]
Btw I am not able to add labels to this issue.

**Comment by micheleriva** [comment]
Hi @28development! I've converted your issue into a discussion; I feel this could be interesting for other devs.

I'd suggest you hosting the Krabs application on a different server so that you can manage your isolated Strapi installation on one server (be a Heroku Dyno, DigitalOcean Droplet, AWS EC2, etc.) and Krabs on another isolated environment.
